frontend/src/app/sessions/session-overview/session-overview.component.html

```diff
@@ -53,7 +53,21 @@
 
     <ng-container matColumnDef="last_seen">
       <th mat-header-cell *matHeaderCellDef>Last seen</th>
-      <td mat-cell *matCellDef="let element">{{ element.last_seen }}</td>
+      <td mat-cell *matCellDef="let element">
+        @if (element.idle_state.available) {
+          @if (element.idle_state.idle_for_minutes! === -1) {
+            Never connected
+          } @else {
+            <app-relative-time
+              [date]="
+                subMinutes(Date.now(), element.idle_state.idle_for_minutes!)
+              "
+            />
+          }
+        } @else {
+          {{ element.idle_state.unavailable_reason! }}
```

Comment:
```suggestion
          {{ element.idle_state.unavailable_reason }}
```

The assertion is not needed here, it works without.